keyboards/unk/info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "keyboard_name": "UNK",
+    "maintainer": "herpiko",
+    "width": 14,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_default": {
+          "layout":[{a:7},"Esc",{a:4},"~\n`","!\n1","@\n2","#\n3","$\n4","%\n5","^\n6","&\n7","*\n8","(\n9",")\n0","_\n-","+\n=",{a:6,w:2},"Backspace"],
```

Comment:
This is incorrect - pass the KLE raw JSON through https://qmk.fm/converter first.